src/observation.c

```diff
@@ -138,7 +139,7 @@ u8 make_propagated_sdiffs_wip(u8 n_local, navigation_measurement_t *m_local,
 
 int sdiff_search_prn(const void *a, const void *b)
```

Comment:
rename to cmp_sid_sdiff()
there are sdiff_sdiff and sdiff_sid functions at the top of the file